### ISSUE TITLE
mek teleporters and chunk loaders should be unbaned and heres why

### ISSUE THREAD

**Issue by Luckyradkid1** [StargateMC/IssueTracker#167]
they are cannon, as seen from season 6 episode 15 of SG1 furlings built a teleporter exactly like the mek tele and it was used to travel offworld, here is a picture:
![image](https://user-images.githubusercontent.com/65505019/83702765-355a1500-a5d3-11ea-9d8a-9e3a9d83a569.png)

also chunkloaders unbanned cause u cant really use the mek teleporters with out it
thx

**Comment by JayCoolGuy** [comment]
Bysokar said chunk loaders put too much strain on the server right now.

**Comment by Joey4213** [comment]
teleporters will not be unbanned as they are un-cannon and we are wanting to relay on stargates or spaceships but mainly stargates for travel so they will not be unbanned, ever... unless you can make one god damn good compelling argument, and @JayCoolGuy is correct chunkloaders in mass put too much strain on the server, think if every player had a chunkloader on maximum power thats atleast 25 chunks loaded 100% of the time on different worlds and as we have about 30 currently logged players thats alot of strain, so if you can pay for a server provider using something closer to a quantum computer then what we currently have then sure, otherwise no.

**Comment by BenFRC5147** [comment]
Teleporters are Iffy I agree, as much as I'd like to use them, but perhaps a mod could be made that allows chunkloaders to be used by patreon subscribers? perhaps each tier gets you x-more chunks to load, and they wouldn't load the chunks on server start, just if the player loaded them and left. This allows Chunkloading for those who want it and gives the server more epic money. Although I do not know if that necessarily violates the EULA of MC.

**Comment by Joey4213** [comment]
It would only violate it as much as something like buycraft, paying for a rank and in turn items, perks, commands.. etc

**Comment by Luckyradkid1** [comment]
joey, they are cannon as i said, but ok

**Comment by DEVensiv** [comment]
@Luckyradkid1 MEk teleporters as is are not canon since they have unlimited range and seemingly 0 power consumption. So As long as you dont have a modified version of mek that integrates with AR and properly calculates the power cost for distance they are not canon. (Furlings only used it to teleport to a mon)

**Comment by TheRedCreations** [comment]
I'm currently developing an Asgard beam mod. This should allow you to travel from your station to the planet below and back. You can also transport between 2 stations if they are in same planet orbit. You also can teleport yourself on the planet between 2 locations

**Comment by StargateMC** [comment]
- Chunk loaders are banned in most cases as they keep worlds loaded that do not need to be. Some loaders which can only hold a chunk loaded for a few hours after a player logs out could be considered I guess, but I have limited time so they arent a priority. 

MC Servers start to die in the ass if they have more than ~200 loaded at once... and forge deliberately unloads additional dimension that arent in use to keep this number as low as possible.

- To update the mekanism teleporter to suit our needs would take me maybe a day (8 hours) of work. Frankly I dont see the benefit when we have a perfectly good space elevator... I'd rather get rings working between orbit/surface instead.

**Comment by JayCoolGuy** [comment]
Yes, there is no real advantage gained from bringing mekanism teleporters back and modifying them. Previously the only real advantage would being able to transport great distances in almost no time and with almost no power.  But if they're going to be modified into something functioning only as a transporter between planet to ship, or planet to moon then there's no point at all.

**Comment by Joey4213** [comment]
I do happen to agree with @JayCoolGuy on this because we already have a decent number of transport between planet and station and planet to planet in same system, there would be no point in adding the mek tele's back because it just add another pointless method

**Comment by Luckyradkid1** [comment]
ok
